docs/user_guide.md

```diff
@@ -422,6 +423,10 @@ analyzer arguments:
                         compilers themselves, and use them with Clang. This is
                         used when the compiler on the system is special, e.g.
                         when doing cross-compilation. (default: False)
+  --capture-analysis-output
+                        Store the (standard and error) output of the
+                        successful analyzer invocations into the
+                        <report-dir>/success directory.
```

Comment:
Update the help here accordingly to the changes in the `help=` of this argument.